types/koa/index.d.ts

```diff
@@ -686,7 +686,7 @@ declare namespace Application {
         request: Request;
     }
 
-    type ParameterizedContext<StateT = any, CustomT = {}> = BaseContext & {
+    interface ExtendableContext extends BaseContext {
```

Comment:
Is this necessary, or would you just augment `BaseContext` instead? It also seems like the purpose of `ParameterizedContext` is to add custom extensions to `BaseContext` (via `CustomT` and `Middleware`):

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f59635899ed15815241c55bc795fe371393b469/types/koa/index.d.ts#L500-L502

Reply:
I only want to add to `ctx`, but `BaseContext` will also add to `context`.